mkdocs.yml

```diff
@@ -33,30 +33,6 @@ extra:
 ## Page tree
 nav:
     - Home: index.md
```

Comment:
The home page is left for now, it will redirect to https://docs.up42.com/sdk
As for the rest of the pages, they are already transferred to
- https://docs.up42.com/sdk/reference
- https://docs.up42.com/sdk/installation
- https://docs.up42.com/sdk/authentication
- https://docs.up42.com/sdk/geometry